pommerman/agents/player_agent.py

```diff
@@ -16,7 +73,7 @@ def __init__(self, agent, control):
                 key.LEFT: 3,
                 key.RIGHT: 4,
                 key.SPACE: 5,
-                key.M: 6 # In Pommerman, this will freeze the game.
+                # key.M: 6 # In Pommerman, this will freeze the game.
```

Comment:
I saw your comment that this crashes the game, however I was not having that issue before your last PR. Can you describe this more? Is there any other behavior that occurs?

Reply:
This was definitely in before, [see for example here](https://github.com/MultiAgentLearning/playground/blob/373fbd2850231b7de6bf2c7315bd6b12c2b46d09/pommerman/utility.py#L46), which is a commit from before I even knew about this project.

Actually, this might be me misunderstanding "freeze" as "crash" where it originally just means "pause" and it worked well all along. I'll put them back in while addressing your other comments.

Reply:
Actually, no, it really is a problem, I get this:

```
Traceback (most recent call last):
  File "/home/beyer/academic/pommerman/venv/bin/pom_battle", line 11, in <module>
    load_entry_point('pommerman', 'console_scripts', 'pom_battle')()
  File "/home/beyer/academic/pommerman/playground/pommerman/cli/run_battle.py", line 118, in main
    run(args)
  File "/home/beyer/academic/pommerman/playground/pommerman/cli/run_battle.py", line 80, in run
    infos.append(_run(seed, record_pngs_dir_, record_json_dir_))
  File "/home/beyer/academic/pommerman/playground/pommerman/cli/run_battle.py", line 60, in _run
    actions = env.act(obs)
  File "/home/beyer/academic/pommerman/playground/pommerman/envs/v0.py", line 115, in act
    return self.model.act(agents, obs, self.action_space)
  File "/home/beyer/academic/pommerman/playground/pommerman/envs/utility.py", line 420, in act
    ret.append(act_ex_communication(agent))
  File "/home/beyer/academic/pommerman/playground/pommerman/envs/utility.py", line 396, in act_ex_communication
    time.sleep(300)
NameError: name 'time' is not defined
```

It's just missing an `import time` in `envs/utility.py`, I can fix that in another PR since it has nothing to do with this one. It does look like a weird feature, though, pausing for 5min without possibility to unpause :laughing:

Reply:
Oh, that must have happened in all the refactoring. Thanks for catching it.

Regarding the pausing, it's actually been pretty darn helpful for me for both testing the game world and for verifying my agents' actions with the game state. We now have more machinery in place to do that (e.g. the game state files), so might not be as vital. 

That being said, adjusting it so that the game continues if you press it again would be great 🥇